docs/collecting-data/collecting-from-own-applications/react-native-tracker/anonymous-tracking/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Anonymous Tracking"
+date: "2022-08-30"
+sidebar_position: 25
+---
+
+# Anonymous Tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+:::info
+
+This feature is available since v1.3.
+
+:::
+
+Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
+
+The affected user and session identifiers are stored in two context entities: [Session](../tracking-events/index.md#session) and [Platform context](../tracking-events/index.md#platform). The Session context entity contains user and session identifiers, while the Platform context entity contains user identifiers. Concretely, the following user and session identifiers can be anonymised:
```

Comment:
I see that this part differs from the other one because we have different links to `Session` and `Platform context`. Some options:
* Don’t link at all, just mention.
* Make the reusable fragment a React component that receives the links as arguments (seems like too much work...).
* Start creating more SDK-independent pages that explain common terminology.
* Abandon the idea of making this a reusable fragment.

Reply:
I changed the links to refer to the schemas on Iglu Central so it should avoid the problem.